docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -287,26 +272,39 @@ And further down in the source, you can use `#if` directives to use those librar
     }
 ```
 
-And that's it!
+Now you can build.
 
-### But What about Portable Class Libraries (PCL)?
+```
+$ dotnet restore
+$ dotnet build
+```
 
-If you want to cross-compile with a PCL target, you'll need to do one more thing before using `#if` directives to conditionally compile different targets: You'll need to add a compilation definition in your `project.json` file in order to target a PCL with #if directives in the source code.
+Your `/bin/Debug` folder will look like this:
 
-For example, if you wanted to target [PCL profile 328](http://embed.plnkr.co/03ck2dCtnJogBKHJ9EjY/preview) (.NET 4.0, Windows 8, Windows Phone Silverlight 8, Windows Phone 8.1, Silverlight 5.0), you may want to refer to it to as "PORTABLE328" when cross-compiling.  Simply add it to the `project.json` file as a `compilationOptions` attribute:
+```
+$ ls bin/Debug
 
+net40/
+net45/
+netstandard1.5/
 ```
+
+### But What about Multitargeting with Portable Class Libraries (PCL)?
```

Comment:
You've already introduced PCL earlier so this title doesn't need to do it again. I'd remove the acronym from here,